comment/models/blocker.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from comment.messages import BlockState
+from comment.managers import BlockedUserManager
+
+
+class BlockedUser(models.Model):
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, blank=True, null=True)
+    email = models.EmailField(blank=True, null=True)
+    blocked = models.BooleanField(default=True)
+
+    objects = BlockedUserManager()
+
+    def __str__(self):
+        return getattr(self.user, self.user.USERNAME_FIELD) if self.user else self.email
+
+
+class BlockedUserHistory(models.Model):
+    UNBLOCKED = 0
+    BLOCKED = 1
```

Comment:
maybe use `enum` here?

Reply:
I prefer the current.

Reply:
at other places in the codebase, we have actually used enum, so it would be better in my opinion to be consistent.

Reply:
also, this may be extended to support a state where users are blocked only for a certain time period, like temporarily blocked.

Reply:
> at other places in the codebase, we have actually used enum, so it would be better in my opinion to be consistent.

I think we have the current one 😃 https://github.com/Radi85/Comment/blob/develop/comment/models/flags.py#L15-L25